### PR TITLE
topology2: cavs-rt5682: Assign RTC_AEC to secondary core (depends on PR 8469)

### PR DESCRIPTION
**Pull request by yongzhi1**
Run RTC_AEC module on core 1 to balance workloads.

### CODE REVIEW

**Comment by yongzhi1** [comment]
Depends on https://github.com/thesofproject/sof/pull/8469.

**Review by lgirdwood — APPROVED** [review]
LGTM.

File: tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake [hunk 1]
```diff
@@ -57,14 +57,14 @@ PDM1_MIC_B_ENABLE=1,DMIC0_PCM_ID=99,PREPROCESS_PLUGINS=nhlt,\
 NHLT_BIN=nhlt-sof-mtl-max98357a-rt5682.bin,DEEPBUFFER_FW_DMA_MS=10,HEADSET_SSP_DAI_INDEX=2,\
 SPK_ID=6,SPEAKER_SSP_DAI_INDEX=0,HEADSET_CODEC_NAME=SSP2-Codec,SPEAKER_CODEC_NAME=SSP0-Codec,\
 BT_NAME=SSP1-BT,BT_INDEX=1,BT_ID=7,BT_PCM_NAME=Bluetooth,INCLUDE_ECHO_REF=true,USE_CHAIN_DMA=true,\
-DEEPBUFFER_D0I3_COMPATIBLE=true,GOOGLE_RTC_AEC_SUPPORT=1"
+DEEPBUFFER_D0I3_COMPATIBLE=true,GOOGLE_RTC_AEC_SUPPORT=1,GOOGLE_AEC_DP_CORE_ID=1"
```
Comment (lkoenig) [inline tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake:60]: Would it be possible to go for Core 2 so nothing else is running on it ?

Comment (yongzhi1) [reply tools/topology/topology2/sof-ace-tplg/tplg-targets.cmake:60]: Ack, let's keep core0 at arm's length :wink:

**Comment by yongzhi1** [comment]
Update:
Move RTC_AEC module to core 2.

**Comment by lgirdwood** [comment]
@lkoenig good for you now ?

**Comment by lkoenig** [comment]
> @lkoenig good for you now ?
If someone did test it on hardware I am good. Otherwise I'll give it a try today.

**Comment by lgirdwood** [comment]
> > @lkoenig good for you now ?
> > If someone did test it on hardware I am good. Otherwise I'll give it a try today.

@yongzhi1 has been testing alongside others.

**Comment by lgirdwood** [comment]
@kv2019i fyi.

**Comment by yongzhi1** [comment]
> @yongzhi1 I added the depends on to the summary, please correct if this no longer holds. I've been holding off merge due to the dependency.

sure, it was tested on mtl-007-drop-stable only, thanks!!

**Comment by lgirdwood** [comment]
@yongzhi1 conflicts.

**Comment by yongzhi1** [comment]
Rebased, thanks!

**Comment by lgirdwood** [comment]
@yongzhi1 any update on dependencies being merged ?

**Comment by yongzhi1** [comment]
> @yongzhi1 any update on dependencies being merged ?

Hi, @lgirdwood @kv2019i think it's OK to merge this patch now as AEC issues were opened any way :)

**Review by kv2019i — APPROVED** [review]

**Pull request merged**